### PR TITLE
Refactor run-msgpush.sh and add virtual machine setup script.

### LINKED ISSUE #1: Details at run-msgpush.sh
The [script](https://github.com/gcinterceptor/java-experiments/blob/master/run-msgpush.sh) when executed here requires that exists a 2instances/ in tmp/. Another detail when executed an experiment with no GCI is that the script still try to catch out the shed files, but they does not exist. 
I also think that at the comments about the expected parameters should be especified the need of a environment variable called USE_GCI and if the GCI should be used, that variable should be set to true.

### PR DESCRIPTION
**Pull request by dfquaresma**
This PR fix #1 and add the customization script used to launch instances used at experiments.

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -0,0 +1,42 @@
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+
+events {
+       worker_connections 768;
+       # multi_accept on;
```
Comment (danielfireman) [inline nginx.conf]: Remove comments, please.

File: nginx.conf [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+http {
+       upstream backends {
+           #server 1.1.1.1:3000 max_fails=0 fail_timeout=15s;
+           #server 1.1.1.2:3000 max_fails=0 fail_timeout=15s;
```
Comment (danielfireman) [inline nginx.conf]: This comment is important, but it needs to come with instructions.

File: run-msgpush.sh [hunk 3]
```diff
@@ -21,7 +22,7 @@ else
 fi
 
 # Experiment configuration
-echo "OUTPUT_DIR: ${OUTPUT_DIR:=/tmp/2instances}"
+echo "OUTPUT_DIR: ${OUTPUT_DIR:=/tmp/2instances}"; mkdir -p "$OUTPUT_DIR"
```
Comment (danielfireman) [inline run-msgpush.sh]: The mkdir needs to go to its own line. It is easier to read and debug.

File: vm-setup.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+# install wrk2.
+apt-get install build-essential libssl-dev git -y
+git clone https://github.com/giltene/wrk2.git
+cd wrk2
+make
+cp wrk /usr/local/bin
+cd ../
+
+# install openjdk.
+apt-get install openjdk-8-jdk -y
```
Comment (danielfireman) [inline vm-setup.sh]: Why installing 8 and 9? And could you please make sure the scripts checks which is being used?

File: nginx.conf [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+       }
+
+       ##
+       # Logging Settings
+       ##
+       log_format exp '$msec;$status;$request_time;$upstream_response_time';
+       access_log /var/log/nginx/access.log exp;
+       error_log /var/log/nginx/error.log;
+
+       ##
```
Comment (danielfireman) [inline nginx.conf]: Remove these unused comments.

File: vm-setup.sh [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+apt-get -o Dpkg::Options::="--force-overwrite" install openjdk-9-jdk -y
+
+# install nginx. Remove the hash character below to do it if needed.
+# apt-get install nginx -y
```
Comment (danielfireman) [inline vm-setup.sh]: Why is this commented?

Comment (dfquaresma) [reply vm-setup.sh]: I thought that was only needed to install nginx at the load balancer VM.

Comment (danielfireman) [reply vm-setup.sh]: That's correct. So please add a new script to setup load balancer vm instead of leaving it here.

File: vm-setup.sh (hunk 4 above)
Comment (dfquaresma) [reply vm-setup.sh]: I had a new idea thanks to you, thanks!! I can install only openjdk9 and be happy. We will use only one, right? I have notice until now, after install a version of a jdk, the jdk being used is the last one installed. We can see it using a "java -version" or "update-alternatives --config java". The last one gives me the option to choose which jdk to use.

File: vm-setup.sh (hunk 6 above)
Comment (dfquaresma) [reply vm-setup.sh]: Working on it...

Comment (dfquaresma) [reply vm-setup.sh]: ... done. :)

**Pull request merged**